packer/conf/buildkite-agent/hooks/environment

```diff
@@ -21,7 +21,7 @@ s3_download() {
   env -i aws s3 cp --quiet ${aws_s3_args[@]} "$1" /dev/stdout
 }
 
-echo "~~~ :house_with_garden: Setting up the environment"
+echo "~~~ Setting up the environment"
```

Comment:
Maybe I should put them back. I've seen people removing them in their own forks, to give their own emojis more prominence! But perhaps I need to apply a CSS b&w filter to emojis that come from `~~~` lines (which are already subdued)